.github/workflows/test.yml

```diff
@@ -23,6 +23,7 @@ jobs:
         run: |
           yarn --ignore-engines
           yarn build
+          yarn lint
```

Comment:
I think this should be a separate `lint` job. That way the failure is clearer (also we don’t need to run it on every Node version + OS)